content/docs/composition-vs-inheritance.md

```diff
@@ -111,9 +111,9 @@ function WelcomeDialog() {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/kkEaOZ?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/gaearon/pen/kkEaOZ?editors=0010)
 
-Composition works equally well for components defined as classes:
+Կոմպոզիցիան աշխատում է հավասարապես լավ կոմպոնենտների համար\` հայտարաված որպես կլասներ.
 
 ```js{10,27-31}
 function Dialog(props) {
```

Comment:
Mars Exploration Program -> Մարսի հետազոտության ծրագիր
How should we refer to you? -> Ինչպե՞ս կարող ենք կապվել ձեզ հետ
Sign Me Up! -> Գրանցեք ինձ
Welcome aboard, ${this.state.login}! -> Բարի՛ գալուստ ${this.state.login}